opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -500,20 +489,11 @@ namespace Opm
 
             printFlowTrailer(mpi_size_, threads, report);
 
-            const auto extraConvOutput = ConvergenceOutputConfiguration {
-                EWOMS_GET_PARAM(TypeTag, std::string, OutputExtraConvergenceInfo),
-                R"(OutputExtraConvergenceInfo (--output-extra-convergence-info))"
-            };
-
-            if (extraConvOutput.want(ConvergenceOutputConfiguration::Option::Steps)) {
-                namespace fs = ::std::filesystem;
-
-                const auto infostep = fs::path { eclState().getIOConfig().getOutputDir() } /
-                    fs::path { eclState().getIOConfig().getBaseName() }.concat(".INFOSTEP");
-
-                std::ofstream os(infostep);
-                report.fullReports(os);
-            }
+            detail::handleExtraConvergenceOutput(report,
+                                                 R"(OutputExtraConvergenceInfo (--output-extra-convergence-info))",
+                                                 EWOMS_GET_PARAM(TypeTag, std::string, OutputExtraConvergenceInfo),
```

Comment:
Looks like you swapped those two lines, thereby interchanging the role of the option value and the option name.

Reply:
... cheeses, not my day.